src/compiler/crystal/tools/doc/html/css/style.css

```diff
@@ -349,6 +349,43 @@ code {
   font-family: Consolas, 'Courier New', Courier, Monaco, monospace;
 }
 
+span.flag {
+  padding: 2px 4px 1px;
+  border-radius: 3px;
+  margin-right: 3px;
+  font-size: 11px;
```

Comment:
I'd suggest folding multiple `border` declarations into:
```css
border: 1px solid transparent;
```
and l8r in `span.flag.<color>`:
```css
border-color: #color;
```